### ISSUE TITLE
5th Learning

### ISSUE THREAD

**Issue by roqhdehd502** [roqhdehd502/Franklin-Detecter#6]
### 진행 전
- 신경망 수정
![NaminNeuralNetwork_mk3](https://user-images.githubusercontent.com/47406388/69895516-a4308a00-1374-11ea-9b59-c6f0597e71d4.jpg)
1. 기존 4층의 레이어를 3층으로 축소
2. 커널 크기 수정
-  이미지 양(Volume) 수정
1. 훈련데이터 증가
2. 시험데이터 축소
### 진행 중
- Tensorflow, numpy, matplotlib Import
- 이미지 파일을 불러온 후 텍스트 파일과 통해 이미지 전처리
- CNN 구현
- Softmax로 확률 구현
- Adam으로 Optimizer 구현
- 배치사이즈와 학습률을 두 가지 경우로 개별 학습
- 훈련 파일을 ckpt 확장자로 저장 후 시험
### 진행 후
- 테스트차 배치(Batch)는 최대, 에폭(Epoch)은 100으로 설정
[6th Learning.txt](https://github.com/roqhdehd502/Franklin-Detecter/files/3906726/6th.Learning.txt)
- 손실율(Loss)은 기존과 차이없었으나 정확도(Acc)가 기존보다 약 17.02%상승
- 이후 배치와 에폭을 조정하면 정확도가 상승할 것으로 기대
